cli/printer/mql.go

```diff
@@ -38,6 +30,18 @@ func (print *Printer) Results(bundle *llx.CodeBundle, results map[string]*llx.Ra
 	return res.String()
 }
 
+// Results prints a full query with all data points
+// NOTE: ensure that results only contains results that match the bundle!
+func (print *Printer) Results(bundle *llx.CodeBundle, results map[string]*llx.RawResult) string {
+	assessment := llx.Results2Assessment(bundle, results)
+
+	if assessment != nil {
+		return print.Assessment(bundle, assessment)
+	}
+
+	return print.Datas(bundle, results)
```

Comment:
just to double check: is this PR just a cleanup or are there functional changes?

from what I see the only thing that happens here is that we've moved the for loop into its own `Datas` func that still gets called if `assessment` is nil

Reply:
ah, i see the use case in https://github.com/mondoohq/cnspec/pull/1438 for extracting that

Reply:
This just splits the function into 2 so that it can be used from cnspec